### DISCUSSION TITLE
OG-Core Developer Meeting, June 5, noon-1pm ET

### DISCUSSION THREAD

**Discussion by rickecon** [PSLmodels/OG-Core#869] [Meeting]
**Agenda for OG-Core developer meeting: 6/5/23**
12:00-1:00pm ET

Join Google Meet video conference
https://meet.google.com/ynf-zpdr-pbx

Join by phone
‪(US) +1 252-394-8749‬ PIN: ‪321 186 757‬#

This meeting is open to the public.

**Agenda**
* Population modeling, new parametric approach
* Adding idiosyncratic income uncertainty
* Calibration of OG-USA and calibration documentation

Next meeting: Monday, June 19, noon-1pm ET

@rickecon @jdebacker @jpycroft @nikhilwoodruff @SeaCelo @sarvonz @prrathi

**Comment by jpycroft** [comment]
Hi Rick,

Apologies, I have another appointment this evening.

Let me know if I can help with anything.

Best, Jon

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Richard Evans ***@***.***>
Sent: Monday, June 5, 2023 8:20:15 AM
To: PSLmodels/OG-Core ***@***.***>
Cc: jpycroft ***@***.***>; Mention ***@***.***>
Subject: [PSLmodels/OG-Core] OG-Core Developer Meeting, June 5, noon-1pm ET (Discussion #869)


Agenda for OG-Core developer meeting: 6/5/23
12:00-1:00pm ET

Join Google Meet video conference
https://meet.google.com/ynf-zpdr-pbx

Join by phone
‪(US) +1 252-394-8749‬ PIN: ‪321 186 757‬#

This meeting is open to the public.

Agenda

  *   Population modeling, new parametric approach
  *   Adding idiosyncratic income uncertainty
  *   Calibration of OG-USA and calibration documentation

Next meeting: Monday, June 19, noon-1pm ET

@rickecon<https://github.com/rickecon> @jdebacker<https://github.com/jdebacker> @jpycroft<https://github.com/jpycroft> @nikhilwoodruff<https://github.com/nikhilwoodruff> @SeaCelo<https://github.com/SeaCelo> @sarvonz<https://github.com/sarvonz> @prrathi<https://github.com/prrathi>
